training/model_evaluation/components/model_prediction/spec.yaml

```diff
@@ -79,7 +78,7 @@ outputs:
     type: mltable
 
 code: ../../src
-environment: azureml://registries/azureml/environments/model-evaluation/versions/4
+environment: azureml://registries/azureml/environments/model-evaluation/versions/3
```

Comment:
why there is difference between environment versions here?

Reply:
in remaining 2 components we are mapping version 4 but in Model Prediction we are mapping to version 3. why?